frontend/src/lib/components/neuron-detail/NnsNeuronRewardStatusAction.svelte

```diff
@@ -9,7 +9,7 @@
     isNeuronFollowingReset,
     isNeuronMissingReward,
     secondsUntilMissingReward,
-    shouldDisplayMissingRewardNotification,
+    isNeuronMissingRewardSoon,
```

Comment:
[nitpick] is it only one reward that they can miss?
```suggestion
    isNeuronMissingRewardsSoon,
```

Reply:
Good point, let me add a TODO.